<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>740</width>
    <height>185</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Century Gothic</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Password Generator</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>icons/main.png</normaloff>icons/main.png</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QPushButton" name="copyingButton">
   <property name="geometry">
    <rect>
     <x>440</x>
     <y>90</y>
     <width>171</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>9</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid;
border-color: #50b347;
border-width: 1px;
border-radius: 20px;
color: 'white';
background-color: rgb(80, 179, 71);
</string>
   </property>
   <property name="text">
    <string>Копировать</string>
   </property>
  </widget>
  <widget class="QSlider" name="lenghtSlider">
   <property name="geometry">
    <rect>
     <x>100</x>
     <y>100</y>
     <width>171</width>
     <height>20</height>
    </rect>
   </property>
   <property name="minimum">
    <number>15</number>
   </property>
   <property name="maximum">
    <number>30</number>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>100</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>10</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>Длина:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="password">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>30</y>
     <width>591</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-style: solid; border-width: 0px; border-radius: 5px;
background-color: rgb(211, 211, 211);
font: 8pt &quot;Century Gothic&quot;;
font-size: 30px;
text-align: center;

</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="maxLength">
    <number>30</number>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="lenght">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>100</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>18</pointsize>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
   <property name="text">
    <string>15</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>90</y>
     <width>92</width>
     <height>52</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QCheckBox" name="digits">
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Цифры</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QCheckBox" name="symbols">
      <property name="font">
       <font>
        <family>Noto Sans</family>
       </font>
      </property>
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Символы</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="generatingButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>30</y>
     <width>101</width>
     <height>101</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Noto Sans</family>
     <pointsize>8</pointsize>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="text">
    <string>Генерировать</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
